Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs

```diff
@@ -33,6 +34,7 @@ public class FileDownloadTest : IntegrationTest
     private const string c_sampleXmlFileName = "SampleXmlFile_06d6ff4d-c124-4d3f-9d96-5e4f2d0c7b0c.xml";
 
     [Test]
+    [IgnoreBrowserLogMessage(@"The file at 'http://.*/res/Remotion.Web.Development.WebTesting.TestSite.Shared/FileDownloadTest.aspx' was loaded over an insecure connection. This file should be served over HTTPS")]
```

Comment:
would it be a good idea to put the messages into a shared const?
(provided they don't go way completely when we disable the log check)